### ISSUE TITLE
code issues

### ISSUE THREAD

**Issue by Herenjia** [bayegaspard/OpenSet-Recognition-for-NIDS#63]
Why can only the SOLE and COOL modes run, while running the other ones indeed requires a hyperparameter configuration file?

**Comment by ambroggi** [comment]
Hello,

It has been a while since I have worked on this project but I may be able to help, can you specify which hyper parameter configuration file you are referring to? I think there are four files that may qualify for that name even if there is only one main one that you are probably talking about. `src/main/Config.py`, `Saves/hyperparam`, `Saves/scoresall.csv`, and `Datasets/hyperparamList.csv`
I don't know of any algorithm we had abbreviated to SOLE. The only one I remember that started with S is softmax, do you mean that?
